### ISSUE TITLE
Alleviate need to edit manually in 0-bootstrap the file for a particular deployment model

### ISSUE THREAD

**Issue by mromascanu123** [terraform-google-modules/terraform-example-foundation#1225]
### TL;DR

In 0-bootstrap the default deployment model is cloudbuild. If not using cloudbuild need to manually rename and edit files. Instead can use a simple script and a hierarchy of deployment-specific configurations (see example image). A simple script can setup the proper environment for a particular deployment model (see prep.sh.zip)
![image](https://github.com/terraform-google-modules/terraform-example-foundation/assets/162719251/95479cdb-85d1-4524-848e-ba31bd6bff40)
[prep.sh.zip](https://github.com/terraform-google-modules/terraform-example-foundation/files/15256324/prep.sh.zip)

### Terraform Resources

```markdown
N/A
```

### Detailed design

```markdown
N/A (see script & tree)
```

**Comment by mromascanu123** [comment]
This was already addressed and fixed in the fork based on TEF and the fix is quite simple - a prep.sh script
Ideally incorporate the fix in TEF - please talk to the Google team working on the fork (Saurajit Banerjee <sabanerjee@google.com>, Brett Tackaberry <tackaberry@google.com>.)

**Comment by eeaton** [comment]
Thanks for flagging @mromascanu123 , can you point me towards this fix in the fork?

**Comment by mromascanu123** [comment]
https://dev.azure.com/ccticei/Migration/_git/TEF-GCP-LZ-HS?version=GBtef_integration_baseline
Please contact the Google team as mentioned
